vendor-extra/ContentPageBundle/src/Resources/config/services.xml

```diff
@@ -8,6 +8,13 @@
 
         <defaults public="false"/>
 
+        <service id="libero.content_page.page.namespace"
+            class="Libero\ContentPageBundle\EventListener\NamespaceListener">
+            <argument>libero</argument>
+            <argument>http://libero.pub</argument>
```

Comment:
This is a bit annoying, but trying to avoid having to redeclare namespaces everywhere.

Reply:
It could be done by the controller when creating the event object rather than being modelled as mutable? Would still need configuration probably, but it may lead to a Factory for the events when we have more than one.